supplements/wild-beyond-the-witchlight.index

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<index>
+    <info>
+        <name>Wild Beyond the Witchlight</name>
+        <update version="0.0.1">
+            <file name="wild-beyond-the-witchlight.index" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/wild-beyond-the-witchlight.index" />
+        </update>
+    </info>
+    <files>
+        <file name="source.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/wild-beyond-the-witchlight/source.xml" />
+        <file name="race-fairy.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/wild-beyond-the-witchlight/race-fairy.xml" />
+        <file name="race-harengon.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/wild-beyond-the-witchlight/race-harengon.xml" />
+        <file name="items.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/wild-beyond-the-witchlight/items.xml" />
+        <file name="backgrounds.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/the-wild-beyond-the-witchlight/backgrounds.xml" />
```

Comment:
also, i know that VRGtR has `backgrounds` as a single file, but i feel like that's a mistake rather than the way we should do it, was there a discussion on this topic? I can't remember one.